commons.webpack.config.js

```diff
@@ -0,0 +1,392 @@
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
this looks like it might have quite some overlap with `webpack.config.js`. Maybe certain things can be imported from there? Or `webpack.config.js` can be adjusted with a few environment variables to output the common-dist artifacts?

Reply:
for now, I just copy & paste from `webpack.config.js`, but ideally we should be able to share the overlapping content.